<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Stutter</class>
 <widget class="QWidget" name="Stutter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="lo">
     <property name="toolTipDuration">
      <number>-2</number>
     </property>
     <property name="title">
      <string>stutter</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="2" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QGroupBox" name="settingsLO">
        <property name="title">
         <string>settings</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_59">
         <item row="0" column="0">
          <widget class="QCheckBox" name="skipLastCheck">
           <property name="text">
            <string>Skip Last Point</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="presetLO">
        <property name="title">
         <string>presets</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_46">
         <item row="5" column="0">
          <widget class="QPushButton" name="normBackTelButton">
           <property name="text">
            <string>Normalized Back Teleport</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="normFrontTelButton">
           <property name="text">
            <string>Normalized Front Teleport</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QPushButton" name="maxBackTelButton">
           <property name="text">
            <string>Max Back Teleport</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QPushButton" name="maxFrontTelButton">
           <property name="text">
            <string>Max Front Teleport</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" rowspan="3">
       <widget class="QGroupBox" name="controlsLO">
        <property name="title">
         <string>controls</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="2" column="0">
          <widget class="QGroupBox" name="aveBpmLO">
           <property name="title">
            <string>average_bpm</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_37">
            <item row="0" column="0">
             <widget class="QDoubleSpinBox" name="aveBpm">
              <property name="minimum">
               <double>0.000000000000000</double>
              </property>
              <property name="maximum">
               <double>100000.000000000000000</double>
              </property>
              <property name="value">
               <double>120.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QGroupBox" name="aveSvLO">
           <property name="title">
            <string>average_sv</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_10">
            <item row="0" column="0">
             <widget class="QDoubleSpinBox" name="aveSv">
              <property name="minimum">
               <double>0.010000000000000</double>
              </property>
              <property name="maximum">
               <double>10.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="2" colspan="2">
          <widget class="QGroupBox" name="typeLO">
           <property name="title">
            <string>type</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_38">
            <item row="1" column="0">
             <widget class="QRadioButton" name="svRadio">
              <property name="text">
               <string>SV</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QRadioButton" name="bpmRadio">
              <property name="text">
               <string>BPM</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="2" colspan="2">
          <widget class="BoxSlider" name="threshold" native="true"/>
         </item>
         <item row="0" column="1">
          <widget class="BoxSlider" name="initSv" native="true"/>
         </item>
         <item row="0" column="0">
          <widget class="BoxSlider" name="initBpm" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0" colspan="4">
       <widget class="QPushButton" name="generateButton">
        <property name="text">
         <string>generate</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="3">
       <widget class="IO" name="input" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="3" rowspan="3">
       <widget class="OutputTP" name="output" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IO</class>
   <extends>QWidget</extends>
   <header>include/io/io.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OutputTP</class>
   <extends>QWidget</extends>
   <header>include/io/outputtp.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BoxSlider</class>
   <extends>QWidget</extends>
   <header>include/boxslider/boxslider.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
